bool request
---
uint8 counted
